### PR TITLE
Add farsi language to locals file

### PR DESCRIPTION
**Pull request by mm580486**

### CODE REVIEW

**Comment by mperham** [comment]
Very cool, first new language in a year or so!

File: web/locales/fa.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# elements like %{queue} are variables and should not be translated
+en: # <---- change this to your locale code
```
Comment (squarism) [inline web/locales/fa.yml]: Should this be `fa`?

**Comment by seuros** [comment]
The issue is that farsi is RTL language.

**Comment by seuros** [comment]
@mm580486 can you should a screenshoot of the dashboard ?

**Comment by squarism** [comment]
RTL = right to left for anyone wondering  🌻

**Pull request merged**